<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>start_window</class>
 <widget class="QMainWindow" name="start_window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1648</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1648</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1648</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_input">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>221</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_output">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Output Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_epsg_num">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>400</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>400</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>EPSG Number</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="margin">
     <number>15</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_epsg_num">
    <property name="geometry">
     <rect>
      <x>442</x>
      <y>230</y>
      <width>1200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1200</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="maxLength">
     <number>20</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Enter EPSG Number like 4326</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_submit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>320</y>
      <width>1200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1200</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_output">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>1200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1200</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Output Folder Path ...</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_input">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>1200</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1200</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1200</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
     </font>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignJustify|Qt::AlignVCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>Input File Name ...</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1648</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
